modules/custom/az_digital_asset_library/config/quickstart/ib_dam.settings.yml

```diff
@@ -1,6 +1,10 @@
 debug: false
-staging: false
+staging: true
```

Comment:
Will this always be true, or should we hold off on marking this as "stable" until this moves out of staging?

Reply:
I am not sure if it will always be true, you can toggle it on and off in the settings to see what all you get from that.